Sources/QuranAudioKit/AudioPlayer/GaplessAudioRequestBuilder.swift

```diff
@@ -68,7 +68,15 @@ final class GaplessAudioRequestBuilder: QuranAudioRequestBuilder {
                     for (offset, verse) in suraTimings.verses.enumerated() {
                         // start from 0 (beginning) if first ayah of the sura
                         let endTime = offset == suraTimings.verses.count - 1 ? suraTimings.endTime : nil
-                        let frame = AudioFrame(startTime: offset == 0 && verse.ayah.ayah == 1 ? 0 : verse.time.seconds, endTime: endTime?.seconds)
+                        
+                        var startTimeSeconds = offset == 0 && verse.ayah.ayah == 1 ? 0 : verse.time.seconds
```

Comment:
Can we merge this logic with the one below it? Something like

if offset == 0 && verse.ayah.ayah == 1 && (requestRuns == .one || ayahs.isEmpty) {
  startTimeSeconds = 0
} else {
  startTimeSeconds = verse.time.seconds
}

Reply:
addressed